pkg/tools/grpcutils/url.go

```diff
@@ -102,3 +102,15 @@ func CheckURLFree(u *url.URL) bool {
 	}
 	return err == nil
 }
+
+// CloneURL clones given url
```

Comment:
@denis-tingaikin 

> 1. I'm not sure that CloneURL should be located in grpcutils.

I agree, both `CloneURL` and `CheckURLFree` are only used in tests, but:
1. I am afraid of creating some general `testutils` package, because it can urgently turn into some very dirty package with lots of different methods.
2. We probably can add `urlutils` package for these, but I am not sure that it would be better than storing them in `grpcutils`.

Any thoughts on this?

> 2. Can we use struts instead pointers to get rid of CloneURL?

Here we need to refactor `nsmgr`, `client`, `registryclient`, `connect` packages to use URL struct as a `connectTo` - it would change API and so affect all deployments without any actual need for them (because it is a test specific case).
We cannot make it on the `sandbox` side, because in all cases this `connectTo` is passed via the non-sandbox options.
So for me it still looks like that using `CloneURL` in tests is the better solution.

Thoughts?

Reply:
Moved to `sandbox`.